### PR TITLE
Footer social media icons valerii

### PR DESCRIPTION
**Pull request by ValDekh**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [x]  Social networks in footer floated (screen in chat), 
- [x]  Text links do not work, icons work
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by ormykhalyshyn — APPROVED** [review]

File: src/app/layout/footer/Footer.styles.scss [hunk 1]
```diff
@@ -73,10 +73,15 @@
     color: c.$pure-white-color;
     @include mut.with-font($font-weight: 300, $font-size: 15px);
     margin-right: 10%;
+    display: flex;
+    align-items: center;
+     height: pxToRem(27px);
```
Comment (Katerix) [inline src/app/layout/footer/Footer.styles.scss:78]: pls remove space (also consider using a mixin for height)

File: src/app/layout/footer/Footer.styles.scss [hunk 2]
```diff
@@ -73,10 +73,15 @@
     color: c.$pure-white-color;
     @include mut.with-font($font-weight: 300, $font-size: 15px);
     margin-right: 10%;
```
Comment (Katerix) [inline src/app/layout/footer/Footer.styles.scss:75]: mixin is also good here

File: src/app/layout/footer/Footer.styles.scss [hunk 3]
```diff
@@ -73,10 +73,15 @@
...
+    display: flex;
+    align-items: center;
```
Comment (Katerix) [inline src/app/layout/footer/Footer.styles.scss:77]: and here ( @include mut.flexed($align-items: center) )

File: src/app/layout/footer/Footer.styles.scss [hunk 4]
```diff
@@ -73,10 +73,15 @@
...
+     height: pxToRem(27px);
     > * {
-      display: inline;
       margin-right: pxToRem(14.62px);
     }
+      li:nth-child(5) {
+        height: 20px;
```
Comment (Katerix) [inline src/app/layout/footer/Footer.styles.scss:83]: => f.pxToRem()

File: src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss [hunk 5]
```diff
@@ -22,8 +22,21 @@
     color: c.$darkish-gray-color;
   }
 
-  > .donateInputContainerWrapper {
-    @include mut.flexed($justify-content: center); 
+   .donateInputContainerWrapper{
+    position: relative;
+    display: flex;
+    justify-content: center;
+    &::before {
```
Comment (Katerix) [inline src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss:29]: line 27 + 28 is equal to deleted line 26

File: src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss [hunk 6]
```diff
@@ -22,8 +22,21 @@
...
+      content: '';
+      position: absolute;
+      bottom: 0;
```
Comment (Katerix) [inline src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss:32]: use mixins

File: src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss [hunk 7]
```diff
@@ -22,8 +22,21 @@
...
+      width: 100%;
```
Comment (Katerix) [inline src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss:33]: same

File: src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss [hunk 8]
```diff
@@ -179,6 +188,12 @@
       line-height: 15px;
     }
 
+    .donateInputContainerWrapper{
```
Comment (Katerix) [inline src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss:191]: space before {

File: src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss [hunk 9]
```diff
@@ -187,12 +202,6 @@
       align-items: center;
       height: 42px;
```
Comment (Katerix) [inline src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss:203]: same

File: src/features/AdminPage/ForFansPage/ForFansPage/ForFansPage.style.scss [hunk 10]
```diff
@@ -43,56 +43,57 @@ $articlesImg: "@assets/images/sources/Articles.png";
...
+    background-color: #303f9f;
+  }
+  
+  .modal button:focus {
+    outline: none;
+  }
+  
+  .modal button:last-child {
+    margin-left: 10px;
+  }
```
Comment (Katerix) [inline src/features/AdminPage/ForFansPage/ForFansPage/ForFansPage.style.scss:98]: please use mixins and variables for COLORS

File: src/features/AdminPage/NewStreetcode/ForFansBlock/ForFansBlock.component.tsx [hunk 11]
```diff
@@ -16,7 +16,6 @@ const ForFansBlock = () => {
     useEffect(() => {
         SourcesApi.getAllNames().then((categ) => setCategoriesSelect(categ)).catch((e) => { });
     }, []);
-
```
Comment (Katerix) [inline src/features/AdminPage/NewStreetcode/ForFansBlock/ForFansBlock.component.tsx:19]: i am not sure if removing the empty line is good here

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss [hunk 12]
```diff
@@ -0,0 +1,43 @@
+@use "@sass/mixins/_utils.mixins.scss" as mut;
+@use "@sass/variables/_variables.colors.scss" as c;
+@use "@sass/_utils.functions.scss" as f;
+@use "@sass/variables/_variables.fonts.scss" as fon;
+
+
+
```
Comment (Katerix) [inline src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:7]: a lot of empty lines

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+.additionalText {
+
```
Comment (Katerix) [inline src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:9]: pls remove line here

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+    display: flex;
+    justify-content:flex-end;
+
+    p {
+        font-family: 'Closer Text' !important;
+
+        a {
+            font-weight: 350;
+            color: #891F16;
+
```
Comment (Katerix) [inline src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:26]: line 25 => color variables 
line 26 => is redundant

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+    text-align: right;
+    @include mut.rem-margined($top: 12px, $bottom: 42px);
+    font-size: f.pxToRem(20px);
+    line-height: f.pxToRem(26px);
+    font-style: italic;
+    display: flex;
+    justify-content:flex-end;
+
+    p {
+        font-family: 'Closer Text' !important;
```
Comment (Katerix) [inline src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:21]: use mixin

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+
+    width: 50%;
+    white-space: pre-line;
+    text-align: right;
+    @include mut.rem-margined($top: 12px, $bottom: 42px);
+    font-size: f.pxToRem(20px);
+    line-height: f.pxToRem(26px);
+    font-style: italic;
+    display: flex;
+    justify-content:flex-end;
```
Comment (Katerix) [inline src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:18]: flex mixin here

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+@media screen and (max-width: 1024px) {
+    .additionalText {
+        width: 100%;
+        @include mut.rem-margined($top: 20px, $left: 10px, $bottom: 20px, $right: 10px );
+    }
+}
+
+@media screen and (max-width: 480px) {
+    .additionalText {
+        font-size: f.pxToRem(14px);
```
Comment (Katerix) [inline src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:40]: no need to use rem for font size

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss [hunk 18]
```diff
@@ -0,0 +1,43 @@
...
+        line-height: f.pxToRem(18.48px);
```
Comment (Katerix) [inline src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:41]: same

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss [hunk 19]
```diff
@@ -0,0 +1,43 @@
...
+            font-weight: 350;
+            color: #891F16;
+
+        }
+    }
+}
+
+@media screen and (max-width: 1024px) {
+    .additionalText {
+        width: 100%;
```
Comment (Katerix) [inline src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:33]: mixin

File: src/features/StreetcodePage/TextBlock/ReadMore/ReadMore.styles.scss [hunk 20]
```diff
@@ -11,8 +11,10 @@
 }
 
 .readMoreContainer {
-    @include mut.rem-margined($top: 35px,$bottom: 50px);
-    @include mut.with-font($font-weight: 500)
+
+    @include mut.rem-margined($top: 35px);
+    @include mut.with-font($font-weight: 500);
+
```
Comment (Katerix) [inline src/features/StreetcodePage/TextBlock/ReadMore/ReadMore.styles.scss:17]: line 14 & 17 redundant

File: src/features/StreetcodePage/TextBlock/ReadMore/ReadMore.styles.scss [hunk 21]
```diff
@@ -33,15 +35,15 @@
 @media screen and (max-width: 1024px){
     .text{
```
Comment (Katerix) [inline src/features/StreetcodePage/TextBlock/ReadMore/ReadMore.styles.scss:36]: space before {

**Review by Katerix — CHANGES_REQUESTED** [review]
Good job! Your progress is noticable. However you should consider some advice:
1. Don't use f.pxToRem() with font or line properties.
2. Please use mixins for sizes, displays and fonts
3. Be attentive to code styling and empty lines. Add them when necessary.
4. Use variables ONLY for colors
clean css is important too :)
good luck

File: src/app/layout/footer/Footer.styles.scss (hunk 3 above)
Comment (ValDekh) [reply src/app/layout/footer/Footer.styles.scss:77]: Done

File: src/app/layout/footer/Footer.styles.scss (hunk 1 above)
Comment (ValDekh) [reply src/app/layout/footer/Footer.styles.scss:78]: Done

File: src/app/layout/footer/Footer.styles.scss (hunk 4 above)
Comment (ValDekh) [reply src/app/layout/footer/Footer.styles.scss:83]: Done

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss (hunk 12 above)
Comment (ValDekh) [reply src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:7]: Done

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss (hunk 15 above)
Comment (ValDekh) [reply src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:21]: done

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss (hunk 14 above)
Comment (ValDekh) [reply src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:26]: done

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss (hunk 19 above)
Comment (ValDekh) [reply src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:33]: done

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss (hunk 18 above)
Comment (ValDekh) [reply src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:41]: done

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss (hunk 17 above)
Comment (ValDekh) [reply src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:40]: done

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss (hunk 13 above)
Comment (ValDekh) [reply src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:9]: done

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss (hunk 16 above)
Comment (ValDekh) [reply src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:18]: done

File: src/features/StreetcodePage/TextBlock/ReadMore/ReadMore.styles.scss (hunk 21 above)
Comment (ValDekh) [reply src/features/StreetcodePage/TextBlock/ReadMore/ReadMore.styles.scss:36]: done

File: src/features/StreetcodePage/TextBlock/ReadMore/ReadMore.styles.scss (hunk 20 above)
Comment (ValDekh) [reply src/features/StreetcodePage/TextBlock/ReadMore/ReadMore.styles.scss:17]: Done

File: src/features/AdminPage/NewStreetcode/ForFansBlock/ForFansBlock.component.tsx (hunk 11 above)
Comment (ValDekh) [reply src/features/AdminPage/NewStreetcode/ForFansBlock/ForFansBlock.component.tsx:19]: done

File: src/features/AdminPage/ForFansPage/ForFansPage/ForFansPage.style.scss (hunk 10 above)
Comment (ValDekh) [reply src/features/AdminPage/ForFansPage/ForFansPage/ForFansPage.style.scss]: not my task (I didn't write this code)

File: src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss (hunk 5 above)
Comment (ValDekh) [reply src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss]: not my task (I didn't write this code)

File: src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss (hunk 6 above)
Comment (ValDekh) [reply src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss]: not my task (I didn't write this code)

File: src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss (hunk 7 above)
Comment (ValDekh) [reply src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss]: not my task (I didn't write this code)

File: src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss (hunk 8 above)
Comment (ValDekh) [reply src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss]: not my task (I didn't write this code)

File: src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss (hunk 9 above)
Comment (ValDekh) [reply src/features/AdditionalPages/SupportUsPage/components/DonationBlock.styles.scss]: not my task (I didn't write this code)

**Pull request closed without merging**